helix-term/src/commands.rs

```diff
@@ -2973,27 +2973,27 @@ fn jumplist_picker(cx: &mut Context) {
         }
     };
 
-    let picker = Picker::new(
-        cx.editor
-            .tree
-            .views()
-            .flat_map(|(view, _)| {
-                view.jumps
-                    .iter()
-                    .map(|(doc_id, selection)| new_meta(view, *doc_id, selection.clone()))
-            })
-            .collect(),
-        (),
-        |cx, meta, action| {
-            cx.editor.switch(meta.id, action);
-            let config = cx.editor.config();
-            let (view, doc) = (view_mut!(cx.editor), doc_mut!(cx.editor, &meta.id));
-            doc.set_selection(view.id, meta.selection.clone());
-            if action.align_view(view, doc.id()) {
-                view.ensure_cursor_in_view_center(doc, config.scrolloff);
-            }
-        },
-    )
+    let mut options: Vec<JumpMeta> = cx
+        .editor
+        .tree
+        .views()
+        .flat_map(|(view, _)| {
+            view.jumps
+                .iter()
+                .map(|(doc_id, selection)| new_meta(view, *doc_id, selection.clone()))
+        })
+        .collect();
+    options.reverse();
```

Comment:
Instead of collecting and then reversing the vec we can update `JumpList::iter` to implement `DoubleEndedIterator` and call `view.jumps.iter().rev()`

Reply:
Thanks.
It definitely improves MR.